prowler/providers/aws/services/ec2/ec2_instance_account_imdsv2_enabled/ec2_instance_account_imdsv2_enabled.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "ec2_instance_account_imdsv2_enabled",
+  "CheckTitle": "EC2 instances should use Instance Metadata Service Version 2 (IMDSv2) to protect against SSRF vulnerabilities, you can set the default instance metadata version at the account level for each AWS Region",
+  "CheckType": [
+    "Data Protection"
+  ],
+  "ServiceName": "ec2",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id",
+  "Severity": "medium",
+  "ResourceType": "AwsEc2Instance",
+  "Description": "Ensure that EC2 instances use Instance Metadata Service Version 2 (IMDSv2) to protect against SSRF vulnerabilities.",
+  "Risk": "EC2 instances that use IMDSv1 are vulnerable to SSRF attacks.",
+  "RelatedUrl": "https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-IMDS-new-instances.html#set-imdsv2-account-defaults",
+  "Remediation": {
+    "Code": {
+      "CLI": "aws ec2 modify-instance-metadata-defaults --region <region> --http-tokens required --http-put-response-hop-limit 2",
```

Comment:
What does the --http-put-response-hop-limit flag do?

Reply:
It is added if your instances will host containers. Documentation says: specify -1 to indicate no preference. When -1 (no preference) is specified, at launch, the value defaults to 2 if the AMI requires IMDSv2; otherwise it defaults to 1.